src/js/engagement_view/src/components/graphDisplay/graphLayout/vizGraphFromLensScope.tsx

```diff
@@ -23,6 +16,34 @@ export const getNodeType = (node: BaseNodeProperties) => {
     return "Unknown Type";
 };
 
+export const summarizeLinks = (links: Link[]): SummaryLink[] => {
+    const mergedLinks: Map<string, Link[]> = new Map();
+    const newLinks: SummaryLink[] = [];
+
+    for (const link of links) {
+        const key = [link.source, link.target];
+        key.sort();
+
+        const sKey = String(key[0]) + String(key[1]);
+
+        const names = mergedLinks.get(sKey) || [];
+
+        names.push(link);
+
+        mergedLinks.set(sKey, names);
+    }
+
+    for (const [key, innerLinks] of mergedLinks.entries()) {
+        newLinks.push({
+            source: innerLinks[0].source,
+            target: innerLinks[0].target,
+            name: innerLinks[0].name,
```

Comment:
Maybe a comment here just explaining that, for now, we're effectively choosing the name and directionality at random. In a future PR we'll specify this behavior.